.github/workflows/test.yml

```diff
@@ -1320,6 +1341,44 @@ jobs:
         if: steps.info.outputs.success == 'true'
         run: echo Ok
 
+  # Upload Turbopack's test results into datadog.
+  upload_test_results:
+    name: Upload Test results
+    needs:
+      - turbopack_rust_test1
```

Comment:
As a starting point, only uploads test results. We may expand this to the benchmark results.